autotst/job/job.py

```diff
@@ -286,6 +284,7 @@ def submit_conformer(self, conformer, restart=False):
                     "It appears that this job has already been run, not running it a second time.")
 
         if restart or not attempted:
+            # Getting the number of processors needed for a child job
             copy_molecule = conformer.rmg_molecule.copy()
```

Comment:
number_of_atoms = `conformer.rmg_molecule.get_num_atoms() - conformer.rmg_molecule.get_num_atoms('H')`.  Don't think we need to copy the rmg molecule here.

Reply:
Yes, I remember you having that in your ThermoJob PR. We should do that, it's less memory